scipy/optimize/direct.c

```diff
@@ -0,0 +1,86 @@
+#include "Python.h"
```

Comment:
How about calling this file `_directmodule.c`, like we do other such files?

Reply:
I haven't looked at this in ages. Personally I find it a lot easier (to read, maintain) to call C code by writing a Cython wrapper that acts as the interface between Python and the underlying C?